### PR TITLE
Correct column alignment in footer

### PR DESCRIPTION
**Pull request by melzreal**
image

Left is the live website, right is the corrected version

### CODE REVIEW

**Comment by siapankina** [comment]
pulled your PR, it still looks off for me with h2 
Also I noticed that locally h2 looks much bigger from when it is life 
Screenshot 2020-11-13 at 11 34 54

changed h2 to h3 , looks much better 
Screenshot 2020-11-13 at 11 39 15

File: unboxed/templates/menus/footer/menu.html [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
+      <a href="{% pageurl item.page %}">  <h2 class="footer-title"> {{ item.title }} </h2></a>
     {% else %}
       <h2 class="footer-title"> {{item.title}}</h2>
     {% endif %}
 
     {% if submenu %}
       {% for subitem in submenu %}
         {% if subitem.url %}
-        <ul class="list-unstyled mb-4">
+        <ul class="list-unstyled mb-4 mt-2 mb-2">
```
Comment (siapankina) [inline unboxed/templates/menus/footer/menu.html:21]: Can you please delete first `mb-4` then ?

**Review by siapankina — CHANGES_REQUESTED** [review]

File: unboxed/templates/menus/footer/menu.html (hunk 1 above)
Comment (melzreal) [reply unboxed/templates/menus/footer/menu.html:21]: good catch

Comment (melzreal) [reply unboxed/templates/menus/footer/menu.html:21]: we can leave it actually, I didn't realise it already was there - i'll take off the mb-2

**Comment by melzreal** [comment]
> pulled your PR, it still looks off for me with h2
> Also I noticed that locally h2 looks much bigger from when it is life
> Screenshot 2020-11-13 at 11 34 54
> 
> changed h2 to h3 , looks much better
> Screenshot 2020-11-13 at 11 39 15

Figma says they are h2s so we'll need to respect that same hierarchy - the col-2 is what's forcing everything to be squished, I'd suggest it might be your server? Could you run collectstatic then relaunch it and see if its changed?



image


image


Also you may already have this ticked, but ensuring "disable cache" is ticked in the network tab is needed for the css updates to show:
image

**Comment by siapankina** [comment]
Yup, stays the same 
well, it might be because of my settings, lets see how it looks live
I will approve it

**Review by siapankina — APPROVED** [review]
Let's sheep it ;)

**Pull request merged**